### DISCUSSION TITLE
Why Berts mechanics should not be allowed in the game!

### DISCUSSION THREAD

**Discussion by IamSooy** [sunflower-land/sunflower-land#2811] [General]
Why Berts mechanics should not be allowed in the game! 

1st: it favors to much to the old player, where only they have the means to race for the the OP items (AUCTION)"

2nd: New players coming each season won't be able to keep up since there will be many things to consider to buy just to keep up the the old player (this also affect Mid players) 

3rd: Personally, I think the team has a bias on it since they are playing the game and mostly they do have good "old" items on them.

4th: In the long run old players or whales will monopolize the "OP" items, eventually the game itself.

**Comment by manuelmt03** [comment]
And another point. "Bert's Odd Obsession" will give utility to many items that shouldn't have it, because they are decorative or wearable without boosts. So the rules of the game are being changed in the middle of the game.
Many players who have been playing for a long time are focused exclusively on boosted NFTs, and are not interested in decorations or other types of collectibles.
And those players are also being hurt by this decision, because they are clearly going to lose the race for IP or Luna's Hat.
